homeassistant/components/vesync/common.py

```diff
@@ -49,3 +59,15 @@ async def async_process_devices(
         _LOGGER.debug("%d VeSync switches found", len(manager.switches))
 
     return devices
+
+
+def is_humidifier(device) -> bool:
+    """Check if the device represents a humidifier."""
+
+    # VeSyncHumid200300S is the base for all humidifiers except VeSyncSuperior6000S.
+    return isinstance(device, VeSyncHumidifierDevice)
+
+
+def has_feature(device, dictionary, attribute):
```

Comment:
Can we add typing to these functions?

Reply:
Done and removed has_feature, it was not being used anywhere.